.github/workflows/main.yml

```
@@ -0,0 +1,33 @@
+name: Cypress Tests
+
+on: push
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      
+      - name: Install npm dependencies
+        run: |
+          npm install
+          npm run install:frontend
+          npm run install:backend
+
+          # Uncommment this block of code if you want to run workflow changes locally using act: https://github.com/nektos/act
+          # - name: Install Cypress Dependencies
+          # run: |
+          #   sudo apt-get update
+          #   sudo apt-get install -y libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+          # env:
+          #   DEBIAN_FRONTEND: noninteractive     
+      
+      - name: Cypress tests
+        uses: cypress-io/github-action@v6
+        with:
+          start: npm run start:frontend, npm run start:server
+          wait-on: 'http://localhost:3000'
+          command: npm run test      
```

```suggestion
          command: npm test      
```

The `test` script is a special one, and so, you can omit the `run` instruction.

For more information about it, I recommend the following content: [Conhecendo os detalhes dos scripts do npm](https://youtu.be/Cxp0CCXdasA).

---

Thanks I've updated this line.